arch/arm64/boot/dts/qcom/sdm636-xiaomi-tulip.dts

```diff
@@ -518,6 +532,18 @@
 
 &tlmm {
 	gpio-reserved-ranges = <8 4>;
+
+	ts_int_active: ts-int-active-state {
+		pins = "gpio67";
+		drive-strength = <8>;
+		bias-pull-up;
+	};
+
+	ts_reset_active: ts-reset-active-state {
+		pins = "gpio66";
+		drive-strength = <8>;
+		bias-pull-up;
+	};
```

Comment:
[tulip.dts.txt](https://github.com/sdm660-mainline/linux/files/11644907/tulip.dts.txt)
in the file above you'll find
```
			ts_mux {

				ts_active {
					phandle = <0x17c>;

					mux {
						pins = "gpio66\0gpio67";
						function = "gpio";
					};

					config {
						pins = "gpio66\0gpio67";
						drive-strength = <0x10>;
						bias-pull-up;
					};
				};

				ts_reset_suspend {
					phandle = <0x17e>;

					mux {
						pins = "gpio66";
						function = "gpio";
					};

					config {
						pins = "gpio66";
						drive-strength = <0x02>;
						bias-pull-down;
					};
				};

				ts_int_suspend {
					phandle = <0x17d>;

					mux {
						pins = "gpio67";
						function = "gpio";
					};

					config {
						pins = "gpio67";
						drive-strength = <0x02>;
						bias-disable;
					};
				};
			};
```

Reply:
Then this definition should have `fuinction = "gpio";` otherwise it's no point of setting electrical properties of the pin. Overall conversion from downstream sould look like
```dts
	touchscreen@62 {
		...
		pinctrl-names = "default", "sleep";
		pinctrl-0 = <&ts_default>;
		pinctrl-1 = <&ts_int_sleep>, <&ts_reset_sleep>;
	};
...
...
&tlmm {
	ts_default: ts-pins-default-state {
		pins = "gpio66", "gpio67";
		function = "gpio";
		drive-strength = <16>;
		bias-pull-up;
	};

	ts_reset_sleep: ts-reset-sleep-state {
		pins = "gpio66";
		function = "gpio";
		drive-strength = <2>;
		bias-pull-down;
	};

	ts_int_sleep: ts-int-sleep-state {
		pins = "gpio67";
		function = "gpio";
		drive-strength = <16>;
		bias-disable;
	};
};
```

Reply:
Done and re-tested on the device